### PR TITLE
Install & use eth2.0-deposit-cli via pip

### PR DESCRIPTION
**Pull request by realerikrani**
https://github.com/ethereum/eth2.0-deposit-cli/issues/178#issuecomment-753007864

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -19,3 +19,4 @@ starlette==0.13.6
 toolz==0.11.1
 urllib3==1.26.2
 uvicorn==0.13.0
+git+https://github.com/ethereum/eth2.0-deposit-cli.git@v1.1.0
```
Comment (tsudmi) [inline requirements.txt]:
```suggestion
https://github.com/ethereum/eth2.0-deposit-cli/archive/v1.1.0.zip
```
This way we don't need to  install git for the docker image.

File: utils/keystore.py [hunk 2]
```diff
@@ -1,167 +0,0 @@
...
-class Keystore(BytesDataclass):
-    """
-    Implement an EIP 2335-compliant keystore. A keystore is a JSON file that
-    stores an encrypted version of a private key under a user-supplied password.
-
-    Ref: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2335.md
-    """
-    crypto: KeystoreCrypto = KeystoreCrypto()
-    uuid: str = ''
-    index: int = 0
```
Comment (tsudmi) [inline utils/keystore.py:87]: `index` and `threshold` are added here on top of the copied `Keystore`. I can make a separate PR which will clean up the copied functionality.

**Review by tsudmi — COMMENTED** [review]
Great approach @realerikrani! A lot of redundant code can be removed thanks to this.

File: utils/keystore.py (hunk 2 above)
Comment (realerikrani) [reply utils/keystore.py:87]: OK, I gave you access to the forked repository (check email for the invite).

Comment (tsudmi) [reply utils/keystore.py:87]: Thanks, I've updated it in [fb3e7b8](https://github.com/stakewise/bls-horcrux/pull/4/commits/fb3e7b8c82bdbf73197ee605b65fe77e6d24d3b0)

**Review by tsudmi — APPROVED** [review]

**Pull request merged**